src/components/form/time-picker.tsx

```diff
@@ -138,6 +140,14 @@ type Props<T extends FieldValues> = UseFormRegisterReturn<string> &
           <option value="AM">AM</option>
           <option value="PM">PM</option>
         </select>
+        <Button
+            type="button"
+            variant="icon"
+            buttonText=""
+            className="ml-auto"
+            icon={faTrashCan}
+            onClick={() => deleteTimePicker()}
+          ></Button>
```

Comment:
I think the plan - according to Josh - was to try and keep the trash button out of the time-picker element, to make sure each individual component as simple (and reusable) as possible. Maybe there is some way the trash icon can be added to the add-more-section?